frontend/src/pages/documents/TheDocument.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <section>
+    <h1> {{ title }}</h1>
+  </section>
+</template>
+
+<script lang="ts">
+export default {
+  props: ['id'],
+  data(): { [prop: string]: any } {
+    return {
+      selectedDocument: null,
+    };
+  },
+  computed: {
+    title(): string {
+      return this.selectedDocument.title;
+    },
+  },
+  created(): void {
+    this.selectedDocument = this.$store.getters['documents/documents'].find(value => value.id === this.id);
```

Comment:
Use vuex module import instead of string getters/actions/mutations - see https://github.com/Yaroslaww-1/cte/blob/develop/frontend/src/pages/documents/documents.page.vue